factory/cryptoComponents.go

```diff
@@ -74,10 +73,9 @@ func NewCryptoComponentsFactory(args CryptoComponentsFactoryArgs) (*cryptoCompon
 		keyGen:                               args.KeyGen,
 		privKey:                              args.PrivKey,
 		activateBLSPubKeyMessageVerification: args.ActivateBLSPubKeyMessageVerification,
+		importDbNoSigCheckFlag:               importDbNoSigCheckFlag,
 	}
-	if args.UseDisabledSigVerifier {
-		ccf.consensusType = disabledSigChecking
-		ccf.keyGen = signing.NewKeyGenerator(disabledCrypto.NewDisabledSuite())
+	if importDbNoSigCheckFlag {
```

Comment:
Is the log below still valid?

Reply:
yes, there are a lot of log.Warns in order to be visible enough when running the node with mock verifier.

Reply:
should have changed the log message though :)